src/components/molecules/TableRow.jsx

```diff
@@ -48,7 +59,10 @@ const TableRow = ({ title, description, createdAt, primary }) => {
       </Col>
       <Col>
         <Button $color="#AF3B3B">
-          <LinkStyled to="/projects/1">Destroy</LinkStyled>
+          <LinkStyled onClick={() => setIsOpen(true)}>Destroy</LinkStyled>
+          <ModalWindow isOpen={isOpen} setIsOpen={setIsOpen}>
+            <ModalContent onCancel={() => setIsOpen(false)} onConfirm={deleteProject} />
+          </ModalWindow>
```

Comment:
Better to write ModalWindow outside of Button
```suggestion
    <Button $color="#AF3B3B">
        <LinkStyled onClick={() => setIsOpen(true)}>Destroy</LinkStyled>
    </Button>
    <ModalWindow isOpen={isOpen} setIsOpen={setIsOpen}>
          <ModalContent onCancel={() => setIsOpen(false)} onConfirm={deleteProject} />
    </ModalWindow>
```